src/pages/TrialMap.page

```diff
@@ -0,0 +1,62 @@
+<!--
+ - Created by drang on 10/19/2018.
+ -->
+
+<apex:page id="TrialMap" standardController="Trial__c" extensions="TrialCoordinatesExtension">
+
+    <head>
+        <title>Simple Map</title>
+        <meta name="viewport" content="initial-scale=1.0"/>
+        <meta charset="utf-8"/>
+        <style>
+      #map {
+        height: 100%;
+      }
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+    </head>
+
+    <body>
+
+    <div id="map"/>
+
+    <script>
+      var startCoordinates = {lat: {!trial.Start_point__Latitude__s}, lng: {!trial.Start_point__Longitude__s}};
```

Comment:
What will happen if page is added to 'new trial' layout?

Reply:
If you are referring to what will happen when the page is used for a Trial without coordinates -  {!trial.Start_point__Latitude__s} and {!trial.Start_point__Longitude__s} return null. I am getting "Uncaught SyntaxError: Unexpected token ," in console when I load the page of a new Trial that has no coordinates. But shouldn't the start/end point fields be required when creating a Trial record?

Reply:
If you are referring to what will happen if we use the page on a new Trial record that has no coordinates - {!trial.Start_point__Latitude__s} and {!trial.Start_point__Longitude__s} return null. I get an "Uncaught SyntaxError: Unexpected token ," in the browser console because of that and the page isn't displayed. But shouldn't the start/end point fields be required when creating a new record?